search/src/test/scala/weco/api/search/services/ImagesErrorsTest.scala

```diff
@@ -0,0 +1,45 @@
+package weco.api.search.services
+
+import weco.api.search.images.ApiImagesTestBase
+
+class ImagesErrorsTest extends ApiImagesTestBase {
```

Comment:
Can we combine these with the existing ImagesErrorsTest? https://github.com/wellcomecollection/catalogue-api/blob/fa3e2486340f868e4959274fccf4f1b241549426/search/src/test/scala/weco/api/search/images/ImagesErrorsTest.scala#L6

Reply:
oops, someone forgot to search for a file before creating it